src/pages/createProfile/CreateProfile.tsx

```diff
@@ -16,6 +16,7 @@ import Complete from "./Complete";
 const CreateProfile: FunctionComponent = () => {
   const theme = useTheme();
   const { data: roles = [] } = useGetRolesQuery();
+  const roleNames = extractRoleNames(roles);
```

Comment:
Previously we were getting an array of strings, but now we get an array of objects that have the `name` inside we need to get. I rather keep the call as pure as possible and modify where we need to vs modifying the response. I am open to other options though, imo this gives us the possibility of using the other 'role_id' attribute or even add more in the future